app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -65,6 +67,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
         $scope.emsCommonModel.openstack_infra_providers_exist = data.openstack_infra_providers_exist;
         $scope.emsCommonModel.default_api_port                = 5000;
         $scope.emsCommonModel.amqp_api_port                   = 5672;
+        $scope.emsCommonModel.hawkular_api_port               = 443;
```

Comment:
@yaacov Should the default hawkular port be 443 or 8443? 
A few lines down it is set o 8443.